src/libcrashreporter-gui/CMakeLists.txt

```diff
@@ -38,5 +44,12 @@ target_link_libraries(crashreporter-gui
         PkgConfig::BREAKPAD
 )
 
+if(WIN32)
+    target_compile_definitions(crashreporter-gui PRIVATE UNICODE _UNICODE)
+endif()
+
 set_target_properties(crashreporter-gui PROPERTIES AUTOUIC ON)
 set_target_properties(crashreporter-gui PROPERTIES AUTOMOC ON)
+
+install(FILES CrashReporter.h DESTINATION ${INCLUDE_INSTALL_DIR}/CrashReporterQt/libcrashreporter-gui)
```

Comment:
You may want to use `PUBLIC_HEADER` instead. That way you can add the header to the right target.

Reply:
What's your concern? I've used `PUBLIC_HEADER` before to install public headers (on non-mac platforms, typically).